coverage-report.sh

```diff
@@ -31,16 +31,18 @@ IGNORE_FILENAME_REGEX="
 log_message() {
   local message=${1}
 
-  # Create a colored log ("\033[33;32m" switches to the green color, and
+  # Create a colored stderr log ("\033[33;32m" switches to the green color, and
   # "\033[0m" switches back).
-  echo -e "\033[33;32m******* ${message} *******\033[0m"
+  echo -e "\033[33;32m******* ${message} *******\033[0m" >&2
```

Comment:
ditto

Reply:
Unfortunately this doens't work:
```
syntax error near unexpected token `&'
```

P.S. I'm wondering whether I should've used Python for this script instead?..